pkgs/by-name/pl/platformsh/update.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl jq common-updater-scripts
+#shellcheck shell=bash
+
+set -eu -o pipefail
+
+version=$(curl -s ${GITHUB_TOKEN:+-u ":$GITHUB_TOKEN"} \
+    curl -s https://api.github.com/repos/platformsh/cli/releases/latest | jq -e -r ".tag_name")
+old_version=$(nix-instantiate --eval -A platformsh.version | jq -e -r)
+
+if [[ $version == "$old_version" ]]; then
+    echo "New version same as old version, nothing to do." >&2
+    exit 0
+fi
```

Comment:
With the JSON approach mentioned above, this shouldn't be necessary anymore.

Reply:
Yes removed